README.md

```diff
@@ -59,6 +59,8 @@ A RegisterClient instance e.g. #<RegistersClient::RegisterClient:0x00007f893c55f
 
 ## Items, entries and records
 
+Items, entries and records are all collections. All of the methods in [Collections](#collections) apply to them. 
+
```

Comment:
I think it would be better to try to talk in Ruby terms as much as possible. Something on the lines of:

> [`ItemCollection`](path/to/item_collection.rb), [`EntryCollection`](path/to/entry_collection.rb) and [`RecordCollection`](path/to/record_collection.rb) are all `Enumerable` and implement the same [Collections](#collections) interface.

Reply:
Agreed.

Reply:
Thanks very much for this edit/suggestion. I've no problem with this at all in principle and this is exactly what's needed, but I just have a couple of questions:

* Is there a big difference between saying: "are all `Enumerable`" and "all include `Enumerable`?". It seems like the first should be lowercase but the second makes more sense in the context of a module(?) called `Enumerable`

* I'm struggling a bit with this sentence:

>They all implement the same [Collections](#collections) interface.

It seems to be referring to the interface, which I understand to be the set of public methods currently described under the header which we've called `Collections`. This makes sense, but since "items, entries, and records" are the actual "collections" (in a Ruby sense) unless I misunderstand, aren't our headers a bit misnamed?

Reply:
> Is there a big difference between saying: "are all `Enumerable`" and "all include `Enumerable`?". It seems like the first should be lowercase but the second makes more sense in the context of a module(?) called `Enumerable`.

The meaning is similar. The first talks about the capabilities, the second talks about how the capabilities are obtained. If you like the second better I don't mind :)

About lowercase, I would agree if "Enumerable" were not in monospace (meaning it's code). When it is as such it should be treated as name (`Enumerable` and `enumerable`) are not the same.

re. Collections: It refers to the `Collections` interface which as per right now is not formalised in the code as a module. I'm not sure I follow your concerns. Let's talk offline.

Reply:
That's fine by me; I just wanted to make sure (as much for myself as anything) that the information in this sentence wasn't ambiguous:

> [`ItemCollection`](path/to/item_collection.rb), [`EntryCollection`](path/to/entry_collection.rb) and [`RecordCollection`](path/to/record_collection.rb) are all `Enumerable` and implement the same [Collections](#collections) interface.

It obviously isn't, so I'm more than happy to just go with this wording 👍